envpool/sokoban/sokoban_node.h

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023-2024 FAR AI
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ENVPOOL_SOKOBAN_SOKOBAN_NODE_H_
+#define ENVPOOL_SOKOBAN_SOKOBAN_NODE_H_
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "astar.h"
+#include "envpool/sokoban/level_loader.h"
+
+namespace sokoban {
+
+class SokobanNode {
+ public:
+  static const std::vector<std::pair<int, int>> kDelta;
```

Comment:
The size of this is fixed so you can make it an `std::array<, 4>` and a `constexpr`, but this is just FYI, it's OK like this.